```
-- Checking for module 'gsettings-qt'
--   Package 'gsettings-qt' not found
-- Checking for module 'gsettings-qt'
--   Package 'gsettings-qt' not found
-- Checking for module 'gsettings-qt'
--   Package 'gsettings-qt' not found
-- Checking for module 'gsettings-qt'
--   Package 'gsettings-qt' not found
CMake Error at /usr/share/cmake-4.2/Modules/FindPkgConfig.cmake:1163 (if):
  if given arguments:

    "NOT" "DEFINED" "__pkg_config_checked_tag" "OR" "__pkg_config_checked_tag" "LESS" "OR" "NOT" "tag_FOUND" "OR" "(" "NOT" "REQUIRED;taglib" "STREQUAL" "" "AND" "NOT" "IMPORTED_TARGET;REQUIRED;taglib" "STREQUAL" "IMPORTED_TARGET;REQUIRED;taglib" ")" "OR" "(" "REQUIRED;taglib" "STREQUAL" "" "AND" "NOT" "IMPORTED_TARGET;REQUIRED;taglib" "STREQUAL" "IMPORTED_TARGET" ")"

  Unknown arguments specified
Call Stack (most recent call first):
  plugins/CdrdaoPlugin/CMakeLists.txt:6 (pkg_check_modules)


-- Configuring incomplete, errors occurred!
Error: module thebeat: Child process exited with code 1
error: Recipe `build` failed with exit code 1
Error: Process completed with exit code 1.

```

--- a/com.vicr123.thebeat.yml
+++ b/com.vicr123.thebeat.yml
@@ -42,17 +42,6 @@ modules:
         url: https://github.com/libcdio/libcdio-paranoia/archive/release-10.2+2.0.2.tar.gz
         sha256: fe35208811f3edfcb79ae7b4ad1984f3188aea42c4c8ba4d009efafb36734a8b
 
-  - name: qcoro
-    buildsystem: cmake-ninja
-    config-opts:
-      - -DCMAKE_BUILD_TYPE=Release
-      - -DBUILD_TESTING=OFF
-      - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
-    sources:
-      - type: archive
-        url: https://github.com/qcoro/qcoro/archive/v0.13.0.tar.gz
-        sha256: 4bff7513c5c8e301b66308df05795043b1792ed16381a484e5c990171b8ff19e
-
   - name: libcontemporary
     buildsystem: cmake-ninja
     config-opts:

